recipes/fastapi-third-party-auth/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set python_min = "3.8" %}
```

Comment:
Unless there is a compelling reason, you should stick with the `python_min` conda-forge uses globally (3.9) unless you require a newer version. If 3.9 is ok, then delete this line.

Reply:
Ok, thanks for the clarification.